src/course/syllabus/apprenticeship/server-side-app/schedule.md

```diff
@@ -2,22 +2,30 @@
 layout: schedule
 schedule:
   monday:
-    - name: Bank holiday!
-      start: 9:45
-      end: 18:00
-  tuesday:
     - name: Express middleware
-      start: 10:00
+      start: 10:10
       end: 13:00
       url: /workshops/express-middleware/
       type: workshop
+    - name: Project intro
+      start: 14:00
+      end: 14:15
+      url: ../project
+    - name: Spike research
+      start: 14:15
+      end: 17:45
+      url: ../spikes
```

Comment:
Would we like to break this time up more?

It feels like 3.5 hours could be a long time for them to be working without a structure suggested for how to spend their time. 

We could break the time into something like: decide a topic, research, get an MVP solution working. We could say this at the top of the spikes file. 

Since they have extended project time maybe it's okay not to give too much direction and leave it up to them to decide how they'd like to structure their time. It might be enough to let the mentors know to check in often and encourage them not to get stuck deciding or researching for too long.